index.html

```diff
@@ -279,11 +281,15 @@ <h2 id="h-tag" class="desktop-multipost">Uber Navigation</h2>
             <li class="small-langli">javascript</li>
           </ul>
           <div>
-            <button type="button" class="seeproject desktop-multipost active">See Project</button>
+            <button id="desktop4" onclick="displayModal(3)" type="button" class="seeproject desktop-multipost active">See Project</button>
           </div>
         </div>
       </div>
     </section>
+    <section>
+      <div id="modal" class="workSection modal"></div>
```

Comment:
- [ Required] When I first open your application there is a small `modal div` that shows up, I think this is the `modal` dialog. I would like to suggest that you hide the modal by default and only show it when the `see project` button is clicked

![Screenshot from 2023-03-10 11-51-15](https://user-images.githubusercontent.com/16392046/224269619-835eaedf-77d0-4db1-932a-2bfe73a67bd7.png)